js/stickynav.js

```diff
@@ -1,46 +1,46 @@
 
 $(document).ready(function() {
-  
+
   // define variables
   var navOffset, scrollPos = 0;
-  
+
   // add utility wrapper elements for positioning
-  $("nav").wrap('<div class="nav-placeholder"></div>');
+  $("nav").wrap('<div class="sticky-combo"></div>');
   $("nav").wrapInner('<div class="nav-inner"></div>');
   $(".nav-inner").wrapInner('<div class="nav-inner-most"></div>');
```

Comment:
These wrappers duplicate the excessive nesting hard coded into the HTML file, so it's actually rendering this:

<img width="369" alt="screen shot 2015-11-24 at 12 14 21 pm" src="https://cloud.githubusercontent.com/assets/15164739/11377779/fc6eb73c-92a4-11e5-83db-d33f1b8055db.png">